docs/user-guide/api-mediation/api-mediation-multi-tenancy.md

```diff
@@ -57,25 +57,115 @@ For static onboarding, make sure that the following parameters are correctly spe
 
 For static onboarding, be sure to use the [Gateway static definition example](#gateway-static-definition-example) presented at the end of this article.
 
-## Establishing a trust relationship between domain Gateways and the Cloud Gateway
-
-The following keytool commands are examples of establishing a trust relationship between domain Gateways and the Cloud Gateway.
-
-- Import the public key certificate of all domain Gateways into the truststore of the Cloud Gateway.
-
-  Use the following example of the keytool command when the domain Gateways running on CA11 and CA32 import the certificate into the Cloud Gateway running on CA31:
-
-  `keytool -import -file keystore/ca11/local_ca/local_ca.cer -alias gateway_ca11 -keystore keystore/ca31/localhost/localhost.truststore.p12`
-
-  `keytool -import -file keystore/ca32/local_ca/local_ca.cer -alias gateway_ca32 -keystore keystore/ca31/localhost/localhost.truststore.p12`
-
-- Import a public key certificate of the Cloud Gateway into the truststore of all domain Gateways.
-
-  Use the following example of the keytool command  when the certificate of the Cloud Gateway running on CA31 is imported into the truststore of the domain Gateways running on CA11 and CA32:
-
-  `keytool -import -file keystore/ca31/local_ca/local_ca.cer -alias gateway_ca31 -keystore keystore/ca11/localhost/localhost.truststore.p12`
-
-  `keytool -import -file keystore/ca31/local_ca/local_ca.cer -alias gateway_ca31 -keystore keystore/ca32/localhost/localhost.truststore.p12`
+## Establishing a trust relationship between Domain API ML and the Central API ML
+
+For routing to work in a MultiTenancy configuration, the Central API ML must trust the Domain API ML, and vice versa for registration.
+In other words it is necessary that the root and, if applicable, intermediate public certificates be shared between Central API ML and Domain API ML. 
```

Comment:
I wouldn't use the "in other words" part, just some clear statement.
maybe as a suggestion:

```suggestion
For routing to work in a MultiTenancy configuration, the Central API Mediation Layer must trust the Domain API Mediation Layers for a successful registration into the Discovery Service component.
The Domain API Mediation Layers must trust the Central API Mediation Layer Gateway to accept routed requests.
The certification chain of the Domain API Mediation Layers must be included in the Central API Mediation Layer's truststore and the certification chain of the Central API Mediation Layer must be included in the Domain Mediation Layer's truststore.
```

On a similar topic, I don't see the "domain" concept being introduced in the article, perhaps there should be an explanation of the meaning.

Reply:
Well, domain API ML is kinda explained in the beginning of the article. But it could be better explained, I agree. But not in this PR, please. This PR is for certificates setup section update.